provider/iot/diagram.py

```diff
@@ -0,0 +1,14 @@
+from shared.diagram import BaseDiagram
+
+
+class IoTDiagram(BaseDiagram):
+    def __init__(self, name: str, filename: str, thing_name: str):
+        """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line (D212)](https://app.codacy.com/gh/Cloud-Architects/cloud-discovery/pullRequest?prid=5655786)